pkg/ipam/schedulerplugin/resync.go

```diff
@@ -278,7 +278,8 @@ func (p *FloatingIPPlugin) syncPodIP(pod *corev1.Pod) error {
 	}
 	keyObj, err := util.FormatKey(pod)
 	if err != nil {
-		return err
+		glog.V(5).Infof("sync pod ip format key: %v", err)
```

Comment:
can we print it with pod name?
```suggestion
		glog.V(5).Infof("sync pod %s/%s ip formatKey with error %v", pod.Namespace, pod.Name, err)
```